assets/airports/kaus.json

```diff
@@ -0,0 +1,4072 @@
+        "VIDAA": ["N29d34.38", "W98d21.17"],
+        "WAILN": ["N29d48.49", "W96d34.28"],
+        "WEEED": ["N29d46.75", "W96d10.91"],
+        "WINDU": ["N31d31.83", "W97d4.95"],
+        "WLEEE": ["N30d2.97", "W96d52.22"],
+        "WLMRT": ["N30d44.45", "W97d51.56"],
+        "XWING": ["N30d16.43", "W97d32.53"],
+        "ZEDKU": ["N30d5.69", "W97d39.31"],
+        "ZENZI": ["N30d17.31", "W96d3.50"]
+	},
+	"runways":[
+		{
+			"name": ["17R", "35L"],
+			"end": [
+				["N30d12.82", "W97d40.76", "541ft"],
+				["N30d10.80", "W97d40.71", "487ft"]
+			],
+			"ils": [true, true],
+			"ils_distance": [25, 25],
+			"glidescope": [3.00, 3.00],
+			"loc_maxDist": [28, 28]
+		},
+		{
+			"name": ["17L", "35R"],
+			"end": [
+				["N30d12.23", "W97d39.47", "491ft"],
+				["N30d10.75", "W97d39.43", "473ft"]
+			],
+			"ils": [true, true],
+			"ils_distance": [25, 25],
+			"glidescope": [3.00, 3.00],
+			"loc_maxDist": [28, 28]
+		}
+	],
+	"sids": {
+		"AEROZ1": {
+			"icao": "AEROZ1",
+			"name": "Aeroz One",
+			"rwy": {
+				"KAUS17L": ["AMUSE"],
+				"KAUS17R": ["AMUSE"],
+				"KAUS35L": ["AMUSE"],
+				"KAUS35R": ["AMUSE"]
+			},
+			"body": ["AEROZ"],
+			"exitPoints": {
+				"SJT": ["SJT"]
+			},
+			"draw": [
+				["AMUSE", "AEROZ", "SJT*"]
+			]
+		},
+		"AUS5": {
+			"icao": "AUS5",
+			"name": "Austin Five",
+			"rwy": {
+				"KAUS17L": ["CWK"],
+				"KAUS17R": ["CWK"],
+				"KAUS35L": ["CWK"],
+				"KAUS35R": ["CWK"]
+			},
+			"exitPoints": {
+				"ABI": ["LLO", "ABI"],
+				"AGJ": ["AGJ"],
+				"JCT": ["JCT"],
+				"LLO": ["LLO"],
+				"SJT": ["LLO", "SJT"],
+				"SAT": ["SAT"],
+				"STV": ["AMUSE", "STV"]
+			},
+			"draw": [
+				["CWK", "AGJ*"],
+				["CWK", "LLO*"],
+				["LLO", "ABI*"],
+				["CWK", "JCT*"],
+				["LLO", "SJT*"],
+				["CWK", "SAT*"],
+				["CWK", "AMUSE", "STV*"]
+			]
+		},
+		"BNDIA2": {
+			"icao": "BNDIA2",
+			"name": "Ben Dia Two",
+			"rwy": {
+				"KAUS17L": ["GARDS"],
+				"KAUS17R": ["GARDS"],
+				"KAUS35L": ["GARDS"],
+				"KAUS35R": ["GARDS"]
+			},
+			"exitPoints": {
+				"BNDIA": ["BNDIA"]
+			},
+			"draw": [
+				["GARDS", "BNDIA"]
+			]
+		},
+		"CWK6": {
+			"icao": "CWK6",
+			"name": "Centex Six",
+			"rwy": {
+				"KAUS17L": ["CWK"],
+				"KAUS17R": ["CWK"],
+				"KAUS35L": ["CWK"],
+        "KAUS35R": ["CWK"]
+			},
+			"exitPoints": {
+				"CLL": ["ILEXY", "CLL"],
+				"ILEXY": ["ILEXY"],
+				"JAYJO": ["JAYJO"],
+				"TNV": ["TNV"],
+				"NAVYS": ["NAVYS"],
+				"ACT": ["ACT"]
+			},
+			"draw": [
+				["CWK", "ILEXY*"],
+				["ILEXY", "CLL*"],
+				["CWK", "JAYJO*"],
+				["CWK", "TNV*"],
+				["CWK", "NAVYS*"],
+				["CWK", "ACT*"]
+			]
+		},
+    "ELOEL2": {
+      "icao": "ELOEL2",
+      "name": "Eloel Two",
+      "rwy": {
+        "KAUS17L": ["WLMRT"],
+        "KAUS17R": ["WLMRT"],
+        "KAUS35L": ["WLMRT"],
+        "KAUS35R": ["WLMRT"]
+      },
+      "body": ["ELOEL"],
+      "exitPoints": {
+        "ABI": ["FORSS", "ABI"],
+        "ELOEL": [],

```

Comment:
A few of the departures were using "ELOEL2 ELOEL" as their route when I checked Flightaware

Reply:
Indeed they do, but since ELOEL is not a transition of the ELOEL2 departure, they'll use the ABI, FORSS or KLNGR transitions after reaching ELOEL. For example, in this screenshot, DAL395 has filed ELOEL2 ELOEL ABI, so it should use the ABI transition of the ELOEL2 departure
![image](https://user-images.githubusercontent.com/19408472/42124403-eabccec4-7c59-11e8-9f5d-0bb11c025dfc.png)